lighthouse-core/lib/lh-error.js

```diff
@@ -376,6 +381,7 @@ const ERRORS = {
   },
 
   // Hey! When adding a new error type, update lighthouse-result.proto too.
+  // Only necessary for runtime errors, which come from artifacts or pageLoadErrors.
```

Comment:
So this error will never be a runtime error, correct? Or it won't while it is a silent audit and not part of performance?

Reply:
i wrote this comment idk how it got here in pauls pr :P

Reply:
"this" error? This comment is a continuation of the previous line, and is referring to when proto must be edited